src/sentry/static/sentry/app/components/onboardingWizard/sidebar.tsx

```diff
@@ -12,16 +12,16 @@ import {t} from 'app/locale';
 import {IconLightning, IconLock, IconCheckmark} from 'app/icons';
 import Tooltip from 'app/components/tooltip';
 import SidebarPanel from 'app/components/sidebar/sidebarPanel';
+import {SidebarPanelProps} from 'app/components/sidebar/types';
```

Comment:
I noticed that we are using `React.ComponentProps <typeof SidebarPanel>`. Please, why did you decide to have the types in a separate file?

Reply:
Hmm, yeah that's not a bad idea :)

Reply:
Okay I actually renamed this to `CommonSidebarProps` since this is shared for more than just panel related things.